base/interactiveutil.jl

```diff
@@ -229,46 +233,15 @@ typesof(args...) = Tuple{map(a->(isa(a,Type) ? Type{a} : typeof(a)), args)...}
 
 gen_call_with_extracted_types(fcn, ex0::Symbol) = Expr(:call, fcn, Meta.quot(ex0))
 function gen_call_with_extracted_types(fcn, ex0)
-    if isa(ex0, Expr) &&
-        any(a->(Meta.isexpr(a, :kw) || Meta.isexpr(a, :parameters)), ex0.args)
```

Comment:
you sure this isn't needed? seems like it could be important for keyword-argument calls